sql/src/test/java/org/apache/druid/sql/calcite/BaseCalciteQueryTest.java

```diff
@@ -1056,7 +1056,7 @@ public Map<String, Object> baseQueryContext()
 
   public void assertResultsEquals(String sql, List<Object[]> expectedResults, List<Object[]> results)
   {
-    for (int i = 0; i < results.size(); i++) {
+    for (int i = 0; i < Math.min(results.size(), expectedResults.size()); i++) {
```

Comment:
totally crazy stream-of-conciousness nit: I don't think that the JVM can optimize the conditional here as the various lists *could* have their sizes change.  If you just use your IDE to extract this as a variable and then use that, it doesn't have to re-evaluate it.

I doubt this would actually show up in profiling as consuming time, but still...

Reply:
totally true :) I was hitting some exception...and was just showeling in something before going back to that